examples/cst/cst_NodeSelector.py

```diff
@@ -18,23 +24,25 @@ class NodeSelector(cst.CSTVisitor):
     """
 
     def __init__(self) -> None:
-        self.pandas_imported: bool = False
-        self.pandas_star_imported: bool = True  # TODO: use this in code
-        self.pandas_aliases = ["pandas"]
-        self.imported_pandas_aliases: list[cst.ImportAlias] = []
 
         self.variables: dict[str, Node] = {}
+        """Maps the library names in namespace to the responsible InputModule"""
+        self.libraries: dict[str, InputModule] = {}
+        """Maps the method names in namespace to the responsible InputModule and the original method name."""
+        self.library_methods: dict[str, Tuple[InputModule, str]] = {}
 
         super().__init__()
 
     def generic_visit(self, cst_node: CSTNode) -> Node:
         class_name = str(cst_node.__class__.__name__).replace("CST", "")
-        method_name = f"visit_{class_name}"
+        method_name = f"parse_{class_name}"
 
         if not hasattr(self, method_name):
-            raise NotImplementedError(
-                f"Function is not defined for: {method_name}"
-            )  # but actually they are all implemented in the base class as empty methods, right?
+            # raise NotImplementedError(
+            #     f'No way to parse CST Node of type "{class_name}"'
+            # )  # but actually they are all implemented in the base class as empty methods, right?
+            print(f'Warning: No way to parse CST Node of type "{class_name}"')
+            return cst_node
```

Comment:
ja war tatsächlich so. Es ist auch eigentlich ganz so krass schlimm wenn das passiert und wir einfach die cst_node zurückgeben. Ich logge das trotzdem mal. In unserem kleinen Codeausschnitt kommt das mit SimpleString ganz häufig vor.